scripts/release.sh

```diff
@@ -3,34 +3,50 @@
 set -e
 
 source ${DAPPER_SOURCE}/scripts/lib/utils
+source ${SCRIPTS_DIR}/lib/debug_functions
+
+### Functions ###
+
+function determine_repo() {
+    origin_url=$(git config --get remote.origin.url)
+    if [[ "$origin_url" =~ ^https:.* ]]; then
+        echo "$origin_url" | cut -f 4 -d'/'
```

Comment:
Well, since the remote URL looks like `https://github.com/submariner-io/submariner` in which case we need to take the 4th field delimited by / and not everything from the 4th field onwards